tests/common/integration_features/test_suppressions_integration.py

```diff
@@ -1052,7 +1052,7 @@ def test_get_policy_level_suppressions(self):
                                                       'resourceId': '/src/BC_AWS_LOGGING_7.tf:aws_cloudtrail.cloudtrail8'}],
                  'checkovPolicyId': 'acme_AWS_1668010000289'}]}
 
-        expected_suppressions = ['BC_AWS_GENERAL_175', 'BC_AWS_LOGGING_2', 'BC_K8S_26']
+        expected_suppressions = ['404088ed-4251-41ac-8dc1-45264af0c461', 'b68013bc-2908-4c9a-969d-f1640d4aca11', '271c1a79-2333-4a12-bf7d-55ec78468b94']
```

Comment:
TBH the previous format looks more readable.
Can you share why is this needed?

Reply:
the previous is wrong, so readability is not a factor here